code/Controllers/CMSMain.php

```diff
@@ -1089,11 +1090,65 @@ public function Breadcrumbs($unlinked = false)
             ]));
         }
 
+        $badge = null;
+        $status = $this->getStatusFlag($record);
+
+        if (count($status) > 0) {
+            $flags = '';
+            foreach ($status as $flag => $flagData) {
+                $flags .= sprintf(
+                    '<span class="badge version-status version-status--%s" %s>%s</span>',
+                    'status-' . Convert::raw2xml($flag),
+                    (isset($flagData['title'])) ? sprintf(' title="%s"', Convert::raw2xml($flagData['title'])) : '',
```

Comment:
This nested line is a little messy, could you assign it to a variable above the sprintf()

Reply:
Updated